docs/reference/docs/bulk.asciidoc

```diff
@@ -197,6 +197,16 @@ size for your particular workload.
 If using the HTTP API, make sure that the client does not send HTTP
 chunks, as this will slow things down.
 
+[float]
+[[bulk-optimistic-concurrency-control]]
+=== Optimistic Concurrency Control
+
+Each index and delete bulk item can include the `if_seq_no` and `if_primary_term`
+parameters in their respective action and meta data lines. These parameters
+allow controlling how these operations will be performed based on the last
+modification to existing documents. See <<optimistic-concurrency-control>> for more details.
+
+
```

Comment:
Can we include a very short example for passing the `if_seq_no` and `if_primary_term` parameters to the bulk API?

Reply:
good question. we don't do it for other parameters, like routing etc. I think we should consistent and adding it to all may be very verbose.

Reply:
+1, good point